tests/cupy_tests/manipulation_tests/test_rearrange.py

```diff
@@ -115,3 +115,105 @@ def test_flipud_2(self, xp, dtype):
     def test_flipud_insufficient_ndim(self, xp, dtype):
         x = testing.shaped_arange((), xp, dtype)
         return xp.flipud(x)
+
+
+@testing.gpu
+class TestFlip(unittest.TestCase):
```

Comment:
chainer supports numpy v1.9, and `flip` function has been introduced since numpy v1.12.0.
In order to skip this when using older version of numpy, could you add `@testing.with_requires` decorator as follows?
```python
@testing.gpu
@testing.with_requires('numpy>=1.12')
class TestFlip(unittest.TestCase):
```